README.md

```diff
@@ -342,6 +344,14 @@ Only torrent statistics are lost during a crash. (Ratio, Total Uploaded & Downlo
 Higher values will reduce disk usage, at the cost of minor stat loss during a crash.
 Consider increasing to 10800 seconds (3 hours) if running thousands of torrents.
 
+### rTorrent tracker scrape patch
```

Comment:
Can you add it to TOC?

Reply:
What do you mean by TOC?

Reply:
https://github.com/crazy-max/docker-rtorrent-rutorrent/blob/9e6dd21e94671e5137045a4b41e7e79227919d44/README.md?plain=1#L24-L47

Reply:
Done.